package.json

```diff
@@ -46,6 +46,7 @@
     "commander": "2.9.0",
     "event-stream": "3.3.4",
     "express": "4.15.2",
+    "express-handlebars": "^3.0.0",
```

Comment:
I mean add it to our current build step. Then compiled index.html could be put in a public folder. This would require stuff like version to be added via js or some other step, since they would be hardcoded on compile. But I think this approach would be better overall, since we compile other handlebars templates with webpack already.